.github/workflows/spm.yml

```diff
@@ -20,4 +20,4 @@ jobs:
     - name: tvOS Build Tests
       run: xcodebuild -scheme BoringSSL-GRPC build -sdk "appletvsimulator" -destination 'platform=tvOS Simulator,name=Apple TV'
     - name: iOS Device Build Tests
-      run: xcodebuild -scheme BoringSSL-GRPC build -sdk 'iphoneos'
+      run: xcodebuild -scheme BoringSSL-GRPC build -sdk 'iphoneos' -destination 'generic/platform=iOS'
```

Comment:
What's the relevance of this change? I see it was also in the abseil change but do you know why it related to the other `main` change?

Also, looks like I need write access.

Reply:
See https://github.com/firebase/boringSSL-SwiftPM/actions/runs/3871473697. I reenabled the Actions in this repo over the weekend after discovering they had been disabled for disuse. 

Access should be fixed as well.

Reply:
Ah, right. I get it. Thanks.